### ISSUE TITLE
DIscuss approach and strategy: Implement basic auth for strong-pm

### ISSUE THREAD

**Issue by chandadharap** [strongloop/strong-pm#40]

**Comment by rmg** [comment]
@chandadharap the original story here is to "discuss basic auth", which I think has happened. What are you expecting me to deliver?

**Comment by chandadharap** [comment]
Implement ssh tunneling ? You can create a separate story or simply edit
the title of this so you still have its history.

On Thu, Jan 8, 2015 at 4:26 PM, Ryan Graham <notifications@github.com>
wrote:

> @chandadharap <https://github.com/chandadharap> the original story here
> is to "discuss basic auth", which I think has happened. What are you
> expecting me to deliver?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strongloop/strong-pm/issues/40#issuecomment-69275099>.
>



-- 
Chanda Dharap
Director, Engineering
@StrongLoop Inc
(510) 304-3924
chanda@strongloop.com

*StrongLoop <http://strongloop.com/>** makes it easy to **develop APIs
<http://strongloop.com/mobile-application-development/loopback/>** in Node,
plus get DevOps capabilities
<http://strongloop.com/node-js-performance/strongops/> like monitoring,
debugging and clustering**.*

**Comment by sam-github** [comment]
After discussion, I think there are two seperate stories:

1. implement http basic auth (effects pm-install, pm, and pmctl, ... and could effect arc)
2. implement ssh tunnelling

Both should be backlogged.

First should be done first, the second later.

@rmg @chandadharap  would you agree?

**Comment by chandadharap** [comment]
agreed.

**Comment by altsang** [comment]
sorry @chandadharap just pointed me to this, had forgotten about it, +1 to #2, most of the communication to strong-pm at least from an Arc perspective would be privileged and "system to system" in the sense that we don't have roles in Arc to prohibit you from doing one function versus another - everyone is considered a super user at this point.

**Comment by rmg** [comment]
Closing. Created #98 and #99 to capture the actionable items from this discussion.
